test/Scoop-Core.Tests.ps1

```diff
@@ -135,12 +135,6 @@ Describe "get_app_name_from_ps1_shim" -Tag 'Scoop' {
         get_app_name_from_ps1_shim "$shim_path" | should -be "mockapp"
     }
 
-    It "returns app name if file exists and is a shim to an app cerca August 2018" -skip:$isUnix {
```

Comment:
Why was this test removed? 🤔

Reply:
`shim` is tested in the above test, and this should be removed (maybe after 2018.08?)

Reply:
If it is not proper to remove it here, I could make a new PR to remove it. I remove it in passing while modifying the following two tests.